pkg/hintrunner/utils/secp_utils.go

```diff
@@ -2,68 +2,83 @@ package utils
 
 import (
 	"fmt"
-	"math/big"
 
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+	"github.com/holiman/uint256"
 )
 
-func getBaseBig() (*big.Int, bool) {
+func getBaseUint256() uint256.Int {
 	// 2**86
-	return new(big.Int).SetString("77371252455336267181195264", 10)
+	return uint256.Int{
+		0,
+		0x400000,
+		0,
+		0,
+	}
 }
 
-func GetSecPBig() (*big.Int, bool) {
+func GetSecPUint256() uint256.Int {
 	// 2**256 - 2**32 - 2**9 - 2**8 - 2**7 - 2**6 - 2**4 - 1
-	return new(big.Int).SetString("115792089237316195423570985008687907853269984665640564039457584007908834671663", 10)
+	return uint256.Int{
+		0xFFFFFFFEFFFFFC2F,
+		0xFFFFFFFFFFFFFFFF,
+		0xFFFFFFFFFFFFFFFF,
+		0xFFFFFFFFFFFFFFFF,
+	}
 }
 
-func SecPPacked(limbs [3]*fp.Element) (*big.Int, error) {
+func SecPPacked(limbs [3]*fp.Element) (uint256.Int, error) {
 	// https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/cairo_secp/secp_utils.py#L28
 
-	baseBig, ok := getBaseBig()
-	if !ok {
-		return nil, fmt.Errorf("getBaseBig failed")
-	}
+	baseUint256 := getBaseUint256()
 
-	packedBig := new(big.Int)
+	packedUint256 := uint256.NewInt(0)
 	for idx, limb := range limbs {
-		limbBig := AsInt(limb)
-		valueToAddBig := new(big.Int).Exp(baseBig, big.NewInt(int64(idx)), nil)
-		valueToAddBig.Mul(valueToAddBig, limbBig)
-		packedBig.Add(packedBig, valueToAddBig)
+		limbUint256, _ := uint256.FromBig(AsInt(limb))
+		valueToAddUint256 := uint256.NewInt(0).Exp(&baseUint256, uint256.NewInt(uint64(int64(idx))))
+		valueToAddUint256.Mul(valueToAddUint256, limbUint256)
+		packedUint256.Add(packedUint256, valueToAddUint256)
```

Comment:
This as a nitpick but I dont see the need to suffix all the variable names with their type, perhaps only limb. I am aware it was like that before, could you please update it.